easy_maps/templatetags/map_tags.py

```diff
@@ -22,6 +22,7 @@ class MapTag(InclusionTag):
     options = Options(
         Argument('lat', resolve=True, required=True),
         Argument('lng', resolve=True, required=True),
+        Argument('setMarker', required=False, default=False),
```

Comment:
Avand in vedere faptul ca e tras in proiect easy-map-ul, why bother with the backwards compatibility?